src/types/offer/interfaces/offer.interfaces.ts

```diff
@@ -21,7 +21,7 @@ export interface Offer extends CommonEntityFields {
   proficiencyLevel: ProficiencyLevelEnum[]
   description: string
   languages: LanguagesEnum[]
-  enrolledUsers: string[]
+  enrolledUsers?: string[]
```

Comment:
Why did you make it optional? Is it related to your PR?

Reply:
it was causing a typescript error in a file that I was editing, so it's partly related to the PR

Reply:
Okay, but don't forget to check all other files that using this fields. Probably ts erros will be in those files because of this optional field.

Reply:
but if the field is optional, it shouldn't cause errors in files that provided it, neither in files that omitted it, should it?

Reply:
fixed the error)